src/tiles/item-image.ts

```diff
@@ -50,12 +52,32 @@ export class ItemImage extends LitElement {
   }
 
   // Templates
-  private get backgroundImageTemplate() {
+  private get ImageTemplate() {
```

Comment:
QUESTION: Is there a reason this starts with a capital letter? Seems like it should be lowercase.

Reply:
fixed